src/common/user.c

```diff
@@ -114,6 +118,34 @@ groupNameFromId(gid_t groupId)
     FUNCTION_TEST_RETURN(STRING, NULL);
 }
 
+#ifdef HAVE_LIBSSH2
```

Comment:
Would be good to comment that we only use this when building with libssh2 currently, at least imv, so people aren't wondering why this seemingly independent thing is under this #ifdef.

Reply:
Will add a comment to note this.